src/databaseService/index.js

```diff
@@ -154,6 +154,26 @@ export const addNewUserData = async (userUID, userData) => {
   }
 }
 
+export const addDiagnoseResponse = async (diagnoseUID, response) => {
+  try {
+    const userUID = AuthenticationService.getCurrentUserUID()
+    const { username } = await get(`users/${userUID}`)
+    const responsePath = `users/${userUID}/requests/${diagnoseUID}/responses`
+
+    const responseData = {
+      answer: response,
+      answeredBy: username,
+      answeredByUID: userUID
+    }
```

Comment:
Can we remove the `answeredBy` field on user responses and query its username by `answeredByUID` when displaying the answers? If we allow the users to change their usernames in the future, this will be a problem...

cc @MPucheta

Reply:
I think that field is used because admin users has nick name @nahudalla

Reply:
Indeed, this is a problem as adminUID do not correlate to nick name. 

We could implement the same logic for all if we create an admin's account with a real username (or change the actual ones to match those usernames)

Reply:
I meant removing the field **only for user responses**. A response _can_ have both fields, but `answeredBy` would only be used for an Admin response. If `answeredBy` is not present, then the username would be loaded using `answeredByUID`.